automatic/palemoon/tools/chocolateyInstall.ps1

```diff
@@ -4,5 +4,16 @@ $url = '{{DownloadUrl}}'
 $url64 = '{{DownloadUrlx64}}'
 $silentArgs = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
 $validExitCodes = @(0)
+$pwd = "$(split-path -parent $MyInvocation.MyCommand.Definition)"
+
```

Comment:
Short: None || readability.

Long: Preference. I always do that in my own work. 3 lines between declaring boring stuff and the actual work. It's one of the things why my code is easier to read than others. But of course that's a matter of preference, and an automatism. Just my handwriting I guess.